generator/internal/genclient/translator/openapi/openapi.go

```diff
@@ -184,19 +184,55 @@ func (t *Translator) makeObjectField(messageName, name string, field *base.Schem
 	if len(field.AllOf) != 0 {
 		return t.makeObjectFieldAllOf(messageName, name, field)
 	}
-	// TODO(#62) - this is an Any or a map<string, T>, needs a TypezID
-	return &genclient.Field{
-		Name:          name,
-		Documentation: field.Description,
-		Typez:         genclient.MESSAGE_TYPE,
-		Optional:      true,
-	}, nil
+	if field.AdditionalProperties != nil && field.AdditionalProperties.IsA() {
+		// This indicates we have a map<K, T> field. In OpenAPI, these are
+		// simply JSON objects, maybe with a restrictive value type.
+		schema, err := field.AdditionalProperties.A.BuildSchema()
+		if err != nil {
+			return nil, fmt.Errorf("cannot build schema for field %s.%s, error=%q", messageName, name, err)
```

Comment:
nit: if you want to return the underlying error we usually wrap those in Go. This can be achieved with `%w`

```suggestion
			return nil, fmt.Errorf("cannot build schema for field %s.%s: %w", messageName, name, err)
```